core/state/database.go

```diff
@@ -170,6 +171,9 @@ type cachingDB struct {
 
 // OpenTrie opens the main account trie at a specific root hash.
 func (db *cachingDB) OpenTrie(root common.Hash) (Trie, error) {
+	if db.triedb.IsVerkle() {
+		return trie.NewVerkleTrie(nil, db.triedb, utils.NewPointCache(), true)
```

Comment:
The root node should not be `nil`, it should be read by the `trieReader` in the database.

Reply:
except the `trieReader` does not exist now, so this is still an open problem.